src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelGene.tsx

```diff
@@ -45,7 +45,7 @@ type TrackPanelGeneProps = {
 };
 
 // TODO: figure out proper gene and transcript track naming conventions
-const GENE_TRACK_ID = 'track:gene-feat';
+const GENE_TRACK_ID = 'track:gene-focus';
```

Comment:
Here and elsewhere.

There's code around that does `trackId.replace('track:', '')`. We should get rid of it. If the backend is sending us track names that start with the word "track", we should remove this prefix [here](https://github.com/Ensembl/ensembl-client/blob/dev/src/shared/state/genome/genomeSlice.ts#L128), and then ask Andres to remove this from the backend.